arch/arm/src/Makefile

```diff
@@ -102,8 +102,9 @@ ifeq ($(CONFIG_ARM_TOOLCHAIN_ARMCLANG),)
   SCRIPT_OPT  = -T
 else
   LIBPATH_OPT = --userlibpath
-  EXTRA_LIBS += arm_vectors.o
   SCRIPT_OPT  = --scatter=
+  EXTRA_LIBS += arm_vectors.o arm_link_steering.o
```

Comment:
remove arm_vectors.o

Reply:
arm_vectors.o must be declared on the command line on cortex-m(use vector array), otherwise the following error will occur:
`Error: L6236E: No section matches selector - no section to be FIRST/LAST.`